apps/cowswap-frontend/src/legacy/hooks/useRefetchPriceCallback.tsx

```diff
@@ -45,7 +45,7 @@ function handleQuoteError({ quoteData, error, addUnsupportedToken }: HandleQuote
 
         // Add token to unsupported token list
         if (unsupportedTokenAddress) {
-          addUnsupportedToken(unsupportedTokenAddress)
+          addUnsupportedToken(quoteData.chainId, unsupportedTokenAddress)
```

Comment:
we use the quote chain id (which might differ from the global state one as stated in the description)